Cargo.toml

```diff
@@ -20,6 +35,7 @@ tower-util = "0.1"
 tower-http-util = "0.1"
 
 [dev-dependencies]
+tokio = "0.1"
```

Comment:
I would put in an explicit patch release as this will doubtfully work w/ tokio 0.1.0